facia-json/src/main/scala/com/gu/facia/client/json/JodaFormat.scala

```diff
@@ -20,14 +21,15 @@ object JodaReads {
 
   implicit val JodaDateTimeReads : Reads[DateTime] = new Reads[DateTime] {
 
-    val dateFormat = ISODateTimeFormat.dateOptionalTimeParser
-    def parse(s: String) = nonFatalCatch[DateTime] opt (DateTime.parse(s, dateFormat))
+    val dateFormat = ISODateTimeFormat.dateTimeParser().withOffsetParsed()
+
+    def parse(s: String) = allCatch[DateTime] opt (DateTime.parse(s, dateFormat))
 
     def reads(json: JsValue): JsResult[DateTime] = json match {
       case JsNumber(d) => JsSuccess(new DateTime(d.toLong))
       case JsString(s) => parse(s) match {
         case Some(d) => JsSuccess(d)
-        case _ => JsError(JsPath(), s"error.expected.jodadate.format ${dateFormat.toString}")
+        case _ => JsError(JsPath(), s"error.unexpected.date.format. Date: '$s'")
```

Comment:
not related neither but a better error message that show the string that cannot be parsed